docs/fundamentals/code-analysis/quality-rules/ca1710.md

```diff
@@ -102,6 +102,28 @@ You can configure whether to exclude indirect base types from the rule. By defau
 dotnet_code_quality.CA1710.exclude_indirect_base_types = false
 ```
 
+### Additional required suffixes
+
+You can provide additional required suffixes or override the behavior of some hardcoded suffixes by adding the following key-value pair to an *.editorconfig* file in your project:
+
+```ini
+dotnet_code_quality.CA1710.additional_required_suffixes = MyClass->Class
```

Comment:
```suggestion
dotnet_code_quality.CA1710.additional_required_suffixes = [types]->[suffix]
```

Reply:
I would suggest `type` instead of `types`, this is a list of `type->suffix`.

Reply:
```suggestion
dotnet_code_quality.CA1710.additional_required_suffixes = [type]->[suffix]
```